code/crates/test/app/src/state.rs

```diff
@@ -50,12 +50,17 @@ pub struct State {
 // order for each node to likely propose different values at
 // each round.
 fn seed_from_address(address: &Address) -> u64 {
-    address.into_inner().chunks(8).fold(0u64, |acc, chunk| {
```

Comment:
Good idea! Maybe we should allow the test framework to trigger such behavior, what do you think? Will think about how to implement that properly